draft-js-mention-plugin/src/index.js

```diff
@@ -14,9 +14,7 @@ import defaultPositionSuggestions from './utils/positionSuggestions';
 const createMentionPlugin = (config = {}) => {
   const defaultTheme = {
     mention: mentionStyles.mention,
-
     mentionSuggestions: mentionSuggestionsStyles.mentionSuggestions,
-
```

Comment:
The spaces where here by intention to make it easy to distinct which component is affected. Maybe a comment would have been better. Can you add comments?